crates/orchestrator/src/client.rs

```diff
@@ -136,12 +139,52 @@ impl OrchestratorClient {
         };
 
         let mut config = self.wait_for_fn_from_orchestrator(f).await;
-
         config.node_index = From::<u16>::from(node_index);
+        // The orchestrator will generate keys for validator if it doesn't load keys from file
+        config.config.my_own_validator_config =
+            ValidatorConfig::<K>::generated_from_seed_indexed(config.seed, config.node_index, 1);
```

Comment:
If the comment is accurate, maybe we should do an "if loading keys from file" check before calling `generated_from_seed_indexed`.

Reply:
This `get_config()` is called in a function `from_file_or_orchestrator` only when the file loading is failed. I'll add a comment there.

Reply:
I see. Sorry to be nit-picking, but I still don't think the comment here is accurate. According to the comment, I would think this function determines whether to load keys from the file, but it's actually the caller's job to determine this. At least the "if" should be "because".